jmix-flowui/flowui/src/main/java/io/jmix/flowui/impl/DialogsImpl.java

```diff
@@ -999,9 +1014,29 @@ public void open() {
             }
             updateProgress(0);
 
-            dialog.open();
+            AtomicReference<Registration> openedRegistration = new AtomicReference<>();
```

Comment:
Same as for message dialog, it's possible to call `build()` several times. Let's convert this variable to class field, so it's re-set if build() is called again.